modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v2/form_fields.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module PdfConstruction
+    module NoticeOfDisagreement::V2
+      class FormFields
+        def veteran_name
+          'F[0].Page_1[0].VeteransFirstName[0]'
+        end
+
+        def veteran_file_number
+          'F[0].Page_1[0].VeteransSocialSecurityNumber_FirstThreeNumbers[0]'
+        end
+
+        def veteran_dob
+          'F[0].Page_1[0].DateSigned[0]'
+        end
+
```

Comment:
Field 4 'If I am not the veteran, my name is...'

Reply:
Or should we just remove all name references here since we'll likely stamp and auto-size?

Reply:
Yeah, I think we should just ignore the names so we can auto-size them. It was an oversight (copy-paste) to leave the `veteran_name` in.